src/Ast/Type/ArrayShapeNode.php

```diff
@@ -13,15 +13,25 @@ class ArrayShapeNode implements TypeNode
 	/** @var ArrayShapeItemNode[] */
 	public $items;
 
-	public function __construct(array $items)
+	/** @var bool */
+	public $sealed;
+
+	public function __construct(array $items, bool $sealed = true)
```

Comment:
@jrmajor this cause error on Rector side, that's sealed true make all changes to add `...` which actually not, see

https://github.com/rectorphp/rector-src/actions/runs/3706328433/jobs/6281403238

we forced pin to use 1.5.0 and the issue disappear

/cc @TomasVotruba @ondrejmirtes

Reply:
Sorry, that's actually a bug in `__toString()`. I'll provide a fix later today.